pkg/apis/shipper/v1alpha1/types.go

```diff
@@ -235,6 +236,14 @@ type ReleaseCondition struct {
 	Message            string                 `json:"message,omitempty"`
 }
 
+type ReleaseClusterReport struct {
```

Comment:
This PR is only concerned with generating reports from CapacityTargets, and that's ok. We do need to think about how the data structure is going to evolve when we start generating reports for InstallationTargets and TrafficTargets.